src/components/Page/Toolbar/Toolbar.jsx

```diff
@@ -67,26 +86,33 @@ export class Toolbar extends Component {
 
     return (
       <div className="toolbar">
-        <div ref={this.brushTopToolbarRef} className={`top-toolbar z-depth-1 valign-wrapper ${boardState.tool === 'brush' ? '' : 'hide'}`}>
+        <div ref={this.brushTopToolbarRef} className={`top-toolbar z-depth-1 valign-wrapper ${boardState.tool === 'brush' || 'rainbow_brush' ? '' : 'hide'}`}>
           <label>Brush Size</label>
           <input type="range" min="2" max="100" value={this.state.brushSize} ref={this.brushSizeRangeRef} onChange={this.onBrushSizeChange} />
         </div>
 
+        <div ref={this.brushTopToolbarRef} className={`top-toolbar z-depth-1 valign-wrapper ${boardState.tool === 'rainbow_brush' ? '' : 'hide'}`}>
+          <label>Color Speed</label>
```

Comment:
Can this be named `Color Change Speed` or `Color Change Rate` instead?

Reply:
Ok why not 😊